test/helpers/ports.go

```diff
@@ -7,6 +7,8 @@ import (
 	"github.com/phayes/freeport"
 )
 
+var l = sync.Mutex{}
```

Comment:
Have we considered naming it like `freePortLock`? It is not inside some structure so the usage is not so clear with the simple name.

Reply:
```suggestion
var freePortLock = sync.Mutex{}
```

Reply:
Makes sense. Renamed it. PTAL.